service/README.md

```diff
@@ -53,17 +53,87 @@ Change the variable for CCD_DEF_CASE_SERVICE_BASE_URL in bin/configurer/utils/fp
+|SCHEDULER_DB_PORT|ENV|(Optional) Port for Scheduler DB, default is 5050|
+|SCHEDULER_DB_NAME|ENV|(Optional) Name for Scheduler DB, default is fpl_scheduler|
+|SCHEDULER_DB_USER|ENV|(Optional) Username for Scheduler DB, default is fpl_scheduler|
+|SCHEDULER_DB_PASSWORD|SECRET|(Optional) Password for scheduler db, default (for local) is fpl_scheduler|
+|idam.s2s-auth.microservice|application.yaml|Name of FPL service when communicating with other Reform services.|
+|idam.s2s-auth.url|ENV|URL of Service responsible for Service to Service authentication (s2s token provider)|
+|idam.s2s-auth.totp_secret|SECRET|One time password code secret required when communicating with s2s service|
+|idam.api.url|ENV|URL to IDAM service used to authenticate requests and obtain further user details.|
+|idam.client.id|ENV|Client identifier used for OAuth2 login via IDAM.|
+|idam.client.redirect_uri|ENV|Redirect URI used for OAuth2 login via IDAM.|
+|idam.client.secret|SECRET|Secret portion of OAuth 2 flow.|
+|auth.idam.client.baseUrl|ENV|Configuration required by auth-checker-library to communicate with IDAM |
+|auth.provider.service.client.baseUrl|ENV|Configuration required by auth-checker-library to communicate with s2s auth service|
+|spring.mail.host|ENV|Mail server host used for robotics (non Gov notify emails|
+|spring.mail.port|ENV|Port of the mail server|
+|spring.mail.properties.mail.smtp.starttls.enable|ENV|Is mail TLS enabled|
+|spring.mail.properties.mail.smtp.ssl.trust|ENV|TLS mail config|
+|spring.security.enabled|ENV|Indicates if spring security should be enabled|
+|spring.security.oauth2.resourceserver.jwt.issuer-uri|ENV|Expected IDAM issuer-uri|
+|spring.security.oauth2.resourceserver.jwt.jwk-set-uri|ENV|Expected IDAM jwk-set-uri|
+|core_case_data.api.url|ENV|CCD Data store API URL|
+|docmosis.tornado.url|ENV|Docmosis Tornado (document generation service) URL|
+|docmosis.tornado.key|SECRET|Docmosis Tornado (document generation service) API key|
+|document_management.url|ENV|URL for service responsible for storing documents|
+|notify.api_key|SECRET|Gov notify API Key for sending emails|
+|gateway.url|ENV|URL for CCD gateway, currently used to prepare URL leading directly to the document, used in notifications|
+|rd_professional.api.url|ENV|URL for Professional Reference Data service|
+|send-letter.url|ENV|URL for service delivering Bulk Print functionality|
+|ccd.ui.base.url|ENV|URL for CCD Web UI, used in notifications|
+|fpl.local_authority_codes_forbidden_case_submission|ENV|(Optional) Local authority code, which will be blocked from submitting applications|
+|fpl.local_authority_email_to_code.mapping|SECRET|Explained below.|
+|fpl.local_authority_code_to_name.mapping|SECRET|Explained below.|
+|fpl.local_authority_code_to_hmcts_court.mapping|SECRET|Explained below.|
+|fpl.local_authority_code_to_cafcass.mapping|SECRET|Explained below.|
+|fpl.local_authority_code_to_shared_inbox.mapping|SECRET|Explained below.|
```

Comment:
They are not really explained below, more of an example given. Would it be possible to add a link to the headers and then you could do something like 
```
[Example below](#local_authority_code_to_hmcts_court)
```

Reply:
I will do when I open separate PR to explain them if it's fine with you :).